modules/update-preparing-conditional.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing_for_updates/updating-cluster-prepare.adoc
+
+:_content-type: PROCEDURE
+[id="update-preparing-conditional_{context}"]
+= Assessing the risk of conditional updates
+
+A conditional update is an update target that is available but not recommended due to a known risk that applies to your cluster.
+The Cluster Version Operator (CVO) periodically queries the OpenShift Update Service (OSUS) for the most recent data about update recommendations, and some potential update targets might have risks associated with them.
+
+These conditional risks are evaluated by the CVO, and if the risk is not applicable to the cluster, then the target version is available as a recommended update path for the cluster.
+If the risk is determined to be applicable, or if for some reason CVO cannot evaluate the risk, then the update target is available to the cluster as a conditional update.
+
+When you encounter a conditional update while trying to update to a target version, you must assess the risk of updating your cluster to this version.
+Generally, if there is no specific need to update to this target version, it is best to wait for a recommended update path from Red Hat.
```

Comment:
I understand the intention to remove "it is best to" and to just state the recommended action, per the RHSSG, but I feel that this is one of those scenarios where we are specifically trying to avoid being prescriptive. 

The entire module is trying to coach the user on how to make their own decision rather than following any sort of formal instruction, and I'm not sure we want to be on the hook for explicitly telling users to not perform a conditional update. Is it possible to keep this or replace it with similar wording?

Reply:
@skopacz1 Considering the intent of the topic, I think it's okay to leave in the "it is best to". Thanks for the background!

Reply:
Awesome, thanks! I'll still reword to avoid "there is"